Source/WebKit/GPUProcess/media/RemoteAudioTrackProxy.h

```diff
@@ -50,22 +49,23 @@ class RemoteAudioTrackProxy final
     : public ThreadSafeRefCounted<RemoteAudioTrackProxy, WTF::DestructionThread::Main>
     , private WebCore::AudioTrackPrivateClient {
 public:
-    static Ref<RemoteAudioTrackProxy> create(GPUConnectionToWebProcess& connectionToWebProcess, TrackPrivateRemoteIdentifier identifier, WebCore::AudioTrackPrivate& trackPrivate, WebCore::MediaPlayerIdentifier mediaPlayerIdentifier)
+    static Ref<RemoteAudioTrackProxy> create(GPUConnectionToWebProcess& connectionToWebProcess, WebCore::AudioTrackPrivate& trackPrivate, WebCore::MediaPlayerIdentifier mediaPlayerIdentifier)
     {
-        return adoptRef(*new RemoteAudioTrackProxy(connectionToWebProcess, identifier, trackPrivate, mediaPlayerIdentifier));
+        return adoptRef(*new RemoteAudioTrackProxy(connectionToWebProcess, trackPrivate, mediaPlayerIdentifier));
     }
 
     virtual ~RemoteAudioTrackProxy();
 
-    TrackPrivateRemoteIdentifier identifier() const { return m_identifier; };
+    WebCore::TrackID id() const { return m_trackPrivate->id(); };
     void setEnabled(bool enabled)
     {
         m_enabled = enabled;
         m_trackPrivate->setEnabled(enabled);
     }
+    bool operator==(const WebCore::AudioTrackPrivate& track) const { return track == m_trackPrivate.get(); }
```

Comment:
Do you really want to check the equality of a `RemoteAudioTrackProxy` and a `AudioTrackPrivate`? It's not clear where you're using this from.

Reply:
I see now where this is being used from. It still seems a little weird, but I guess it's ok.

Reply:
I thought that it made the code more elegant where it's used